docs/4.0. Behaviour.md

```diff
@@ -0,0 +1,12 @@
+# Behaviour
+
+_(c) AMWA 2019, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
+
+IS-10 defines behaviour for:
+
+- [Authorization Servers](4.1. Behaviour - Authorization Servers.md)
```

Comment:
I'm pretty sure these links won't work Pete. You'll need to URL encode them (e.g. `3.2.%20Discovery%20-%20Peer%20to%20Peer%20Operation.md` such as in https://raw.githubusercontent.com/AMWA-TV/nmos-discovery-registration/v1.3.x/docs/1.0.%20Overview.md)

Reply:
Strange, as I definitely did make that change but perhaps it didn't make it to the PR. Anyway, it should be there now.